examples/gopherlouse 2.mapping

```diff
@@ -0,0 +1,10 @@
+p26:h10
```

Comment:
Just reset this file to the one in master -- https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

Reply:
Removed.